internal/resource/config/datastore/ldap_data_store.go

```diff
@@ -271,11 +271,11 @@ func toStateLdapDataStore(con context.Context, ldapDataStore *client.LdapDataSto
 	var diags, allDiags diag.Diagnostics
 
 	if ldapDataStore != nil {
-		diags.AddError("Failed to read Custom data store from PingFederate.", "The response from PingFederate was nil.")
+		diags.AddError("Failed to read Ldap data store from PingFederate.", "The response from PingFederate was nil.")
```

Comment:
Need to return here, otherwise we will still panic below. diags.AddError doesn't end the method execution.

Reply:
return added